includes/class-wc-payments.php

```diff
@@ -1923,7 +1923,6 @@ public static function add_wcpay_options_to_woocommerce_permissions_list( $permi
 				'woocommerce_remind_me_later_todo_tasks',
 				'woocommerce_deleted_todo_tasks',
 				'wcpay_fraud_protection_welcome_tour_dismissed',
-				'wcpay_payouts_rename_notice_dismissed',
```

Comment:
Non-blocking: I was wondering if there is a way to clean up this option from the wp_options of the merchant website DB to keep it clean. Something using the [delete_option](https://developer.wordpress.org/reference/functions/delete_option/)